src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_ARRAYLIST_SIZE = 15;
+    private int listSize;
+    private int lastRecord = 0;
```

Comment:
```suggestion
    private int size;
```
this variable can be easily used as index for last element